.eslintrc

```diff
@@ -12,7 +12,12 @@
   "rules": {
     "max-len": "off", // Remove warnings on max line length exceeding 100 characters
     "react/require-default-props": "off", // Disabled the requirement to default all non-required props
-    "no-multi-assign": "off" // TODO: Decide if we want to turn this on. It was turned on in eslint-config-airbnb-base v14.1.0
+    "no-multi-assign": "off", // TODO: Decide if we want to turn this on. It was turned on in eslint-config-airbnb-base v14.1.0
+    "jsx-a11y/anchor-is-valid": [ "error", {
+      "components": [ "Link" ],
+      "specialLink": [ "to", "hrefLeft", "hrefRight" ],
+      "aspects": [ "noHref", "invalidHref", "preferButton" ]
+    }]
```

Comment:
is it the same as terra-core

Reply:
It's not part of [terra-core](https://github.com/cerner/terra-core/blob/master/.eslintrc), however I did update eslint, see:

[updated eslint in terra-consumer](https://github.com/cerner/terra-consumer/blob/2465d5542b39e91dee02783d44d8b8ab4bfc90d9/package.json#L85-L89)
vs
[eslint in terra-core](https://github.com/cerner/terra-core/blob/master/package.json#L85-L89)

Reply:
Found this solution [here](https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/339#issuecomment-343878382)

Reply:
IMO we should focus this PR on the react 16 update and possible do the eslint update in a new PR. Thoughts?

Reply:
I updated both ESLint and Stylelint at @alex-bezek's suggestion.

Reply:
this will probably be the only jira we have funded to get into this repo and update. Wanted to make sure to leave as clean as possible

Reply:
if you wanted,  you could make a separate PR. But if that means its a pain for you to strip out the changes, i'm ok with looking through this

Reply:
These changes look good to me knowing your using react-router in this project and that's what this is working around.